package.json

```diff
@@ -59,13 +59,15 @@
     "eslint-plugin-unused-imports": "^1.0.0",
     "esm": "^3.2.25",
     "husky": "^6.0.0",
+    "jest": "^26.6.3",
     "lint-staged": "^10.5.3",
     "npm-run-all": "^4.1.5",
     "only-allow": "^1.0.0",
     "prettier": "^2.1.2",
     "ts-node": "^9.1.1",
     "ttypescript": "^1.5.12",
-    "typescript": "4.1.3"
+    "typescript": "4.1.3",
+    "webcrypto": "^0.1.1"
```

Comment:
why add to workspace root?

Reply:
It's required by https://github.com/DimensionDev/Maskbook/blob/5aa9aeade85b2dc4198075467010f3654e622a47/scripts/jest-setup.js#L1-L14

Reply:
CC @Jack-Works

Reply:
Please move those tests into the `packages/maskbook`

Reply:
They are in `packages/maskbook`.